src/ops/layer_norm_cpu.cc

```diff
@@ -16,7 +16,7 @@ namespace ctranslate2 {
       size_t batch_size = input.size() / depth;
       StorageView tmp({batch_size, depth}, input.dtype(), input.device());
       #pragma omp parallel for
-      for (size_t i = 0; i < batch_size; ++i) {
+      for (long long i = 0; i < batch_size; ++i) {
```

Comment:
Could `unsigned long long` be used? Otherwise, just try to fix the warning by using `unsigned` or casting the limit index.

Reply:
The index has to be signed.